examples/durable-functions/src/functions/hello_world.rs

```diff
@@ -1,18 +1,29 @@
 use azure_functions::{bindings::DurableOrchestrationContext, durable::OrchestrationOutput, func};
-use futures::future::join_all;
+use log::warn;
+use serde_json::Value;
 
 #[func]
 pub async fn hello_world(mut context: DurableOrchestrationContext) -> OrchestrationOutput {
```

Comment:
I noticed you switched to `RefCell` for accessing state. Do we still need to require `mut context` for functions?

Reply:
That's a good question.  Of the top of my head, I don't think so, so this is something we should fix.

Reply:
I double checked this last night.  We no longer need mut and the context methods should be able to take `&self`.